packages/teams-js/src/public/media.ts

```diff
@@ -232,23 +254,26 @@ export namespace media {
       function handleGetMediaCallbackRequest(mediaResult: MediaResult): void {
         if (callback) {
           if (mediaResult && mediaResult.error) {
-            /* eslint-disable-next-line strict-null-checks/all */ /* Fix tracked by 5730662 */
-            callback(mediaResult.error, null);
+            callback(mediaResult.error, new Blob());
           } else {
             if (mediaResult && mediaResult.mediaChunk) {
               // If the chunksequence number is less than equal to 0 implies EOF
               // create file/blob when all chunks have arrived and we get 0/-1 as chunksequence number
               if (mediaResult.mediaChunk.chunkSequence <= 0) {
                 const file = createFile(helper.assembleAttachment, helper.mediaMimeType);
-                callback(mediaResult.error, file);
+                callback(mediaResult.error, file ?? new Blob());
               } else {
                 // Keep pushing chunks into assemble attachment
-                const assemble: AssembleAttachment = decodeAttachment(mediaResult.mediaChunk, helper.mediaMimeType);
-                helper.assembleAttachment.push(assemble);
+                const assemble: AssembleAttachment | null = decodeAttachment(
```

Comment:
>decodeAttachment [](http://example.com/codeflow?start=60&length=16)

`decodeAttachment` can now return `null` (before it was returning `undefined`, see the comments in `mediaUtil.ts`), this handles this error case by not trying to push it into the  `assembleAttachment` array (note that previous to this, it was unconditionally pushing `undefined` into that same array in the error case, which was incorrect and not be handled by any later code)